Files/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -710,22 +714,18 @@ public async Task<ReturnResult> RenameAsync(IStorageItemWithPath source, string
 
             if(source.ItemType == FilesystemItemType.File && Path.GetExtension(source.Path) != Path.GetExtension(newName))
 			{
-                var dialog = new MessageDialog("If you change a file extension, the file might become unusable." 
-                                                + Environment.NewLine +
-                                                "Are you sure you want to change it?", "Rename");
+                var renameDialogText = resourceLoader.GetString("RenameFileDialog/Text");
 
-                dialog.Commands.Add(new UICommand("Yes", async delegate (IUICommand command)
-                {
+                var yesSelected = await DialogDisplayHelper.ShowDialogAsync("Rename", renameDialogText, "Yes", "No");
+                if(yesSelected)
+				{
```

Comment:
Fix formatting, please.

Reply:
Sorry but I have no idea why it looks like that on GitHub, looks fine in VS and VS code..
![image](https://user-images.githubusercontent.com/16183422/106587135-57aa9500-659d-11eb-858c-b19832447866.png)

Can you clone it and check in your editor?

Reply:
Just press Ctrl + K + D in this file.

Reply:
Done.

Reply:
Set the next params and press Ctrl + K, Ctrl + D again.
![image](https://user-images.githubusercontent.com/20501502/106588094-0fff1b80-6553-11eb-86a1-1b21605ee580.png)

Reply:
Done